ios/safari_extension/Makefile

```
@@ -0,0 +1,11 @@
+TARGET := iphone:clang:latest:15.0
+
```

We could add

```suggestion
INSTALL_TARGET_PROCESSES = @@PROJECTNAME@@

```

to make sure the extension is killed on install (Safari will trigger restarting it I’d assume, but best if you test whether this works as expected)